Sounds/wwise_project/ChatCambrioleur.chgalvan.validationcache

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ValidationCache Version="1" SchemaVersion="103">
+	<ValidationCacheItem File="C:\ChatCambrioleur\Sounds\wwise_project\Virtual Acoustics\Factory Acoustic Textures.wwu" DateLow="1639586020" DateHigh="0"/>
```

Comment:
Same as the other audio source files. Is this a hardcoded path?

Reply:
This shouldn't have been checked in, I'll delete it